packages/amplify-migration-tests/src/__tests__/migration_tests_v10/custom-stack.migration.test.ts

```diff
@@ -25,7 +25,7 @@ describe('adding custom resources migration test', () => {
   });
 
   afterEach(async () => {
-    await deleteProject(projRoot);
+    await deleteProject(projRoot, null, true);
```

Comment:
deleteProject changed from `sendConfirmYes` to `sendYes` which hanged on 10.5.1
this will call into latest code.